config/index.js

```diff
@@ -48,7 +48,8 @@ const config = {
       environment: process.env.ROLLBAR_ENV
     },
     urlRequests: {
-      albumList: 'https://jsonplaceholder.typicode.com/albums'
+      albumList: 'https://jsonplaceholder.typicode.com/albums',
```

Comment:
Maybe you could add a baseUrl and then add the corresponding path according if we want photos or albums.